src/components/autocomplete/index.tsx

```diff
@@ -74,6 +79,11 @@ const Autocomplete = ({
   optionsRef,
   optionsContainerRef,
   isFromTransactions,
+  actionOnFocus,
```

Comment:
add default value to params: actionOnFocus, actionOnRemoveInput, actionOnSelect, actionOnBlur
example: `actionOnFocus: () => {}`